localstack/services/apigateway/apigateway_listener.py

```diff
@@ -37,34 +37,34 @@ def forward_request(self, method, path, data, headers):
             search_match = re.search(regex2, path)
             api_id = search_match.group(1)
             stage = search_match.group(2)
-            relative_path = '/%s' % search_match.group(3)
-            try:
-                integration = aws_stack.get_apigateway_integration(api_id, method, path=relative_path)
-                assert integration
-            except Exception:
-                try:
-                    integration = aws_stack.get_apigateway_integration(api_id, 'ANY', path=relative_path)
-                    assert integration
-                except Exception:
-                    # if we have no exact match, try to find an API resource that contains path parameters
-                    path_map = get_rest_api_paths(rest_api_id=api_id)
-                    try:
-                        extracted_path, resource = get_resource_for_path(path=relative_path, path_map=path_map)
-                    except Exception:
-                        return make_error('Unable to find path %s' % path, 404)
-
-                    integrations = resource.get('resourceMethods', {})
-                    integration = integrations.get(method, {})
-                    if not integration:
-                        integration = integrations.get('ANY', {})
-                    integration = integration.get('methodIntegration')
-                    if not integration:
+            relative_path_w_query_params = '/%s' % search_match.group(3)
 
-                        if method == 'OPTIONS' and 'Origin' in headers:
-                            # default to returning CORS headers if this is an OPTIONS request
-                            return get_cors_response(headers)
+            relative_path, query_string_params = extract_query_string_params(path=relative_path_w_query_params)
 
-                        return make_error('Unable to find integration for path %s' % path, 404)
+            path_map = get_rest_api_paths(rest_api_id=api_id)
+            try:
+                extracted_path, resource = get_resource_for_path(path=relative_path, path_map=path_map)
+            except Exception:
+                return make_error('Unable to find path %s (%s, %s, %s)' % (
```

Comment:
Looks like we're missing some `%s` placeholders in the template string.

Also, maybe we can squeeze the into two lines instead of 7:
```
return make_error('Unable to find path %s (%s, %s, %s, %s, %s)' % (
    path, api_id, stage, relative_path_w_query_params, relative_path, query_string_params), 404)
```

Reply:
This is debugging stuff I forgot to remove.  I'll revert it.